src/main/kotlin/com/bloock/sdk/proof/entity/Proof.kt

```diff
@@ -6,6 +6,7 @@ import kotlin.math.floor
 class Proof(
     val leaves: List<String>,
     val nodes: List<String>,
+    val networks: List<String>,
```

Comment:
Networks are not just Strings, its an object containing:
- name: string
- state: string
- txHash: string

Please, review how the JS SDK is implemented.